media/js/base/core-datalayer-init.js

```diff
@@ -4,14 +4,31 @@
 
 // init core dataLayer object and push into dataLayer
 $(function() {
-    if (Mozilla && Mozilla.Analytics) {
-        var dataLayer = window.dataLayer = window.dataLayer || [];
+    var dataLayer = window.dataLayer = window.dataLayer || [];
```

Comment:
If you're comfortable with it being removed, that is fine with me. I just wasn't sure if you were concerned about there being a dependency on another file that is obscure. I will remove it though.